packages/fcl/src/current-user/exec-service/strategies/http-post.js

```diff
@@ -26,25 +26,40 @@ export const getExecHttpPost = (execLocal) => async({service, body, config, opts
   } else if (resp.status === "REDIRECT") {
     return resp
   } else if (resp.status === "PENDING") {
+
+    // these two flags are required to run polling one more time before it stops
     var canContinue = true
+    var shouldContinue = true
+
     const [_, unmount] = await execLocal(
       normalizeLocalView(resp.local),
       {
         serviceEndpoint,
-        onClose: () => (canContinue = false)
+        onClose: () => (shouldContinue = false)
       }
     )
 
     const close = () => {
       try {
         unmount()
-        canContinue = false
+        shouldContinue = false
       } catch (error) {
         console.error("Frame Close Error", error)
       }
     }
+    /**
```

Comment:
This mechanic is a bit confusing to me, could we perhaps rename the variables `canCondinue` `shouldContinue` to include more info on their purpose?

Reply:
I've added comments explaining the flow